### PR TITLE
Extracts content_loader from dmutils

### PR DESCRIPTION
**Pull request by Wynndow**
Copies the main content_loader file as well as associated tests.

Copies two methods from dmutils.config that content_loader depends on and adds them to converters.py. Updates the test file for them as they were previously tested implicitly.

Moves two methods from dmutils.formats as well as tests. Front end apps will need updating to import from here.

### CODE REVIEW

File: .cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (allait) [inline .cache/v/cache/lastfailed]: Let's drop this file

File: dmcontent/converters.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+from flask._compat import string_types
```
Comment (allait) [inline dmcontent/converters.py]: We could change this to `six.string_types`, and then we don't need Flask as a dependency (need `six` though).

File: tests/conftest.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import pytest
+from flask import Flask
+
+
+@pytest.fixture
+def app():
```
Comment (allait) [inline tests/conftest.py]: We don't need this fixture anymore

File: requirements_for_test.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+-r requirements.txt
+Flask==0.10.1
```
Comment (allait) [inline requirements_for_test.txt]: Don't need this anymore

File: scripts/run_tests.sh [hunk 5]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```
Comment (allait) [inline scripts/run_tests.sh]: Let's replace this script with `test` Makefile tasks, like we did in the apps

Comment (Wynndow) [inline scripts/run_tests.sh]: All done!

**Comment by allait** [comment]
Looks great 👍

**Pull request merged**